### PR TITLE
Jj tabbed interface

### PR DESCRIPTION
**Pull request by jessjno**
added user liked photos and user photos links to user profile

### CODE REVIEW

File: app/views/photos/_photo.html.erb [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+  <div class="col-md-6 offset-md-3">
+    <div class="card">
+      <%= image_tag photo.image, class: "card-img-top" %>
+      <div class="card-body">
+        <h5 class="card-title"><%= photo.owner.username %></h5>
+        <p class="card-text"><%= photo.caption %></p>
+      </div>
+      <ul class="list-group list-group-flush">
+        <% photo.comments.each do |comment| %>
+          <li class="list-group-item">
```
Comment (armstrca) [inline app/views/photos/_photo.html.erb:11]: The bullet points next to each comment are rendering outside the parent box - they should be inside

**Review by armstrca — CHANGES_REQUESTED** [review]

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -2,7 +2,31 @@
   <%= @user.username %>
 </h1>
 
-<h2>Own photos</h2>
+<div class="row">
+  <div class="col md-6 offset-3">
```
Comment (armstrca) [inline app/views/users/show.html.erb]: Should be `col-md-6`

File: app/views/users/liked.html.erb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div class="col md-6 offset-3">
```
Comment (armstrca) [inline app/views/users/liked.html.erb]: should be `col-md-6`

**Review by Testell — APPROVED** [review]
Resolved styling looks great!

**Review by armstrca — APPROVED** [review]

**Comment by dgarcia13-99** [comment]
You did a great job formatting the tabbed interface cards, and your styling looks great after the edits!

**Review by dgarcia13-99 — COMMENTED** [review]
Great job making the appropriate changes!

**Pull request merged**